posthog/caching/test/test_warming.py

```diff
@@ -48,15 +48,15 @@ def setUp(self) -> None:
     @patch("posthog.hogql_queries.query_cache.QueryCacheManager.get_stale_insights")
     def test_priority_insights_no_stale_insights(self, mock_get_stale_insights):
         mock_get_stale_insights.return_value = []
-        insights = list(priority_insights(self.team))
+        insights = list(insights_to_cache(self.team))
```

Comment:
`insights_to_cache` feels a bit misleading, since we cache almost all insights. The ones here are the ones where we keep the cache warm. Maybe `insights_to_keep_fresh` or `team_warm_cache_targets`?